mfx/optimize/align.py

```diff
@@ -1,52 +1,96 @@
+from typing import Literal
+from pydantic import validate_call, ConfigDict
 from bluesky import RunEngine
 
 
+Diagnostics = Literal["xcs1", "dg1", "dg2"]
+Methods = Literal["xopt", "blop"]
+Devices = Literal["yag", "wave8"]
+Turbo = Literal["safety", "optimize"]
+
+
+def validate_w_lowercase_args(func):
+    """
+   Decorator to make string inputs lowercase before validating.
```

Comment:
Nitpick: docstring indentation matching
Suggested rewording to make it more clear that this calls a validator

```suggestion
    Decorator to make string inputs lowercase, and then validate.
```